dlp/tests/system/gapic/v2/test_system_dlp_service_v2_vpcsc.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+import pytest
+
+from google.cloud import dlp_v2
+from google.cloud.dlp_v2 import enums
+from google.cloud.dlp_v2.proto import dlp_pb2
+from google.api_core import exceptions
+
+
+PROJECT_INSIDE = os.environ.get("PROJECT_ID", None)
+PROJECT_OUTSIDE = os.environ.get(
+    "GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_PROJECT", None
+)
+IS_INSIDE_VPCSC = os.environ.get("GOOGLE_CLOUD_TESTS_IN_VPCSC", "true")
```

Comment:
This should probably be 'false' by default.

See monitoring's setup
https://github.com/googleapis/google-cloud-python/blob/f890d2916e67e40dfbdfeb814d6b92fce9103f5b/monitoring/tests/system/test_vpcsc.py#L30-L36